validator/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+ name = "validator"
```

Comment:
WDYT:

```suggestion
 name = "linkerd-network-validator"
```

Reply:
Yeah! Sounds better, validator felt a bit vague to me :)